lib/matplotlib/patches.py

```diff
@@ -2562,7 +2562,7 @@ def get_y(self):
         return self._y
 
     def get_width(self):
-        "Return the width of the  rectangle"
+        "Return the width of the rectangle"
```

Comment:
Add a final period ? As you seem to have added it to other docstrings (at least one) :smiling_imp:.